### PR TITLE
Add sandbox support to Bulk

### PR DESCRIPTION
**Pull request by Jeremydavisvt**
Adds sandbox variables to login, and optional sandbox variable to SalesforceBulk()

### CODE REVIEW

**Comment by markgarg** [comment]
Anyone wants to merge this in? This fixes issue #26

File: salesforce_bulk/salesforce_bulk.py [hunk 1]
```diff
@@ -50,14 +50,14 @@ def __init__(self, job_id, batch_id, state_message):
 
 class SalesforceBulk(object):
 
-    def __init__(self, sessionId=None, host=None, username=None, password=None,
+    def __init__(self, sessionId=None, host=None, username=None, sandbox=False, password=None,
```
Comment (lambacck) [inline salesforce_bulk/salesforce_bulk.py]: changes arg order. New arg should be added to the end.

**Comment by lambacck** [comment]
This breaks anyone doing:
```py
bulk = SalesforceBulk(None, hostname, username, password)
```

I'd be willing to merge if sandbox parameter was moved to the end of the constructor args list.

**Comment by Jeremydavisvt** [comment]
You're absolutely right, my bad. I've moved the variable to the end.

**Comment by bholagabbar** [comment]
@mnaberez Could you merge this and push this into deployment/update on pip? I really need this feature on a project

**Pull request merged**